framework/sessions/__init__.py

```diff
@@ -77,6 +76,7 @@ def prepare_private_key():
 
 
 def get_session():
+    Session = apps.get_model('osf.Session')
```

Comment:
```
    from website.app import init_app
  File "/Users/matt/Devel/osf.io/website/app.py", line 23, in <module>
    from framework.sentry import sentry
```
This actually appears to be the cause. If we localize the sentry import this should go away.,